src/components/dialog/dialog.scss

```diff
@@ -249,8 +249,7 @@
   background-color: var(--sbb-dialog-background-color);
   border-block-start: var(--sbb-dialog-footer-border);
 
-  :host(:not([data-slot-names~='title'], [title-content])) &,
-  :host(:not([data-slot-names~='action-group'])) & {
+  :host(:not([data-slot-names~='title'], [title-content], [data-slot-names~='action-group'])) & {
```

Comment:
It seems that with :not() it is an "and" connection between selectors. So the last change probably broke chromatic. One approach would be using :where() or leave it as it was :)

MDN says: `You can negate several selectors at the same time. Example: :not(.foo, .bar) is equivalent to :not(.foo):not(.bar).`

Reply:
yea, stupid error :/